### PR TITLE
Added intro to ux strategy content

### PR DESCRIPTION
**Pull request by hloniphile22**

### CODE REVIEW

**Review by kaleem99 — APPROVED** [review]

File: index.md [hunk 1]
```diff
@@ -1,14 +1,14 @@
 ---
-_db_id: 66
```
Comment (sheenarbw) [inline index.md]: why did you remove the _db_id?

**Review by sheenarbw — CHANGES_REQUESTED** [review]
Please dont every remove the db_id. It's quite important

**Review by sheenarbw — APPROVED** [review]
